package.json

```diff
@@ -32,7 +32,7 @@
   },
   "license": "Apache-2.0",
   "engines": {
-    "node": ">=10.0.0"
+    "node": ">=12.0.0 <15.0.0 || >=16.0.0"
```

Comment:
this also allows to run on node v13.x which we do not support. why this change from recommended on the parent ticket `"node": "^12 || ^14 || ^16"` ?